eq-author-api/schema/resolversTests/validation.test.js

```diff
@@ -244,6 +245,30 @@ describe("resolvers", () => {
       });
     });
 
+    it("can update inclusive and previous answer min values", async () => {
+      const previousAnswer = await createNewAnswer(firstPage, "Number");
```

Comment:
I think this is wrong. We restrict it on the content picker to only type.

Reply:
From the business rule (as implemented in the content picker) we restrict the previous answers to being the same type. This is setting the min value of a currency to a number.